docs/source/markdown/podman-create.1.md.in

```diff
@@ -316,7 +316,8 @@ Suppress output information when pulling images
 
 #### **--rm**
 
-Automatically remove the container when it exits. The default is *false*.
+Automatically remove the container and any anonymous unnamed volume associated with
+the container when it exits. The default is **false**.
```

Comment:
Ideally this should really be included with the @@option mechanism. So both podman-create and podman-run only need to write it once.
But I won't block this over it, the change is better so I am good to merge